doc/user/bgp.rst

```diff
@@ -2567,6 +2567,22 @@ address-family:
 Ethernet Virtual Network - EVPN
 -------------------------------
 
+Note: When using EVPN features and if you have a large number of hosts, make
+sure to adjust the size of the arp  neighbor cache to avoid neighbor table
+overflow and/or excessive garbage collection. On Linux, the size of the table
+and garbage collection frequency can be controlled via the following
+sysctl configurations:
```

Comment:
Could we also throw in a mention of `man 7 arp`?  The kernel GC values are documented there, seems like it would be a nice thing to include as a reference.